third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -3079,9 +3095,18 @@ static void SendMessageToFrontend(const v8_inspector::StringView& message) {
   v8::HandleScope scope(isolate);
 
   v8::Local<v8::Context> context = isolate->GetCurrentContext();
-  v8::Local<v8::Value> arg = v8::String::NewFromTwoByte(isolate, message.characters16(),
+  v8::Local<v8::Value> arg;
+  if (message.length() > v8::String::kMaxLength) {
+    // Print a record replay log message of the first few characters of
+    // messages that are too long to send to the frontend.
+    recordreplay::Print("SendMessageToFrontend CDP message too long: %.*s",
+      100, message.characters16());
+    arg = v8::Undefined(isolate);
```

Comment:
Will this make us throw at https://github.com/replayio/chromium/blob/master/third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc#L221 ? I'm guessing that is where `message` ends up? 

It might be fine to just `return;` here and at https://github.com/replayio/chromium/blob/ddfa34a5d8eaca399690dec14bf29ce5bed12ba4/third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc#L198 seems like we should do
```
   gCurrentMessageId = id;
+gCurrentMessageResult = undefined;
```
then `sendMessage` will just return `undefined` if this case is hit.

Though I do wonder if it'd be better to throw an exception at https://github.com/replayio/chromium/blob/master/third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc#L207 in that case.